templates/Dockerfile-env

```diff
@@ -1,33 +0,0 @@
-FFMPEG_VERSION=%%FFMPEG_VERSION%% \
```

Comment:
not sure what the intent here, it's nice to have one file to parse to see the versions being built.

That avoid disseminating version number and sha256sum in the scripts

Reply:
Specifying the version before the assembly itself gives a significant increase in speed with the parallel assembly of all variants and versions.

When all versions are declared at the beginning of the file, you have to recompile the same libraries again and again.

In my version, you collect library data for the variant only 1 time and many times re-release individual versions of ffmpeg itself.
And instead of 6 hours, I collected all versions in just 40 minutes.

But I agree, this is not very convenient. I propose to accept these changes now, and a bit later I want to redo the templating to jinja2 in order to add variability of the choice of the library version depending on the version of ffmpeg.

Reply:
right , that's the plan, @GnaphronG already started on it : https://github.com/jrottenberg/ffmpeg/tree/refactor/templates/config

there's hope. 

;-)